x-pack/plugin/sql/qa/src/main/resources/constant-keyword.csv-spec

```diff
@@ -0,0 +1,131 @@
+// To mute tests follow example in file: example.csv-spec
+
+//
+// Tests testing field alias (introduced in ES 6.4)
+//
+
+// filtering
+
+filterEquals
+SELECT extra_gender g FROM "test_emp_copy" WHERE g = 'M' LIMIT 5;
+
+    g     
+---------------
+M              
+M              
+M              
+M              
+M   
+
+;
+
+filterNotEquals
+SELECT COUNT(*) AS c FROM "test_emp_copy" WHERE extra_gender <> 'M';
+
+    c:l     
+---------------
+0
+;
+
+aggWithNullFilter
+SELECT COUNT(*) count FROM test_emp_copy WHERE extra_gender IS NOT NULL;
```

Comment:
constant_keyword cannot be null, correct?

Reply:
Theoretically, I think it can be null... haven't tested this.

Reply:
I guess any type can be `null` (missing field in the doc)?

Reply:
For `null` case, all the documents in the index, for that field, shouldn't have a value. As soon as one document is indexed with something in it for that field, all other documents will "inherit" that value.

Reply:
https://github.com/elastic/elasticsearch/issues/53545